pkg/bloombuild/planner/queue/queue.go

```diff
@@ -152,8 +291,10 @@ func (q *Queue) TotalPending() (total int) {
 	return total
 }
 
-func getTaskPath(task Task) string {
-	return filepath.Join("tasks", task.Tenant(), task.Table(), task.ID())
+func getTaskPath(task *protos.ProtoTask) string {
+	table := protos.FromProtoDayTableToDayTable(task.Table)
+	taskFile := task.Id + ".task"
```

Comment:
I would use `.protobuf` extension to indicate that the content is a binary encoded protobuf object